### ISSUE TITLE
HMA1 is not running, even with the demo data

### ISSUE THREAD

**Issue by elenagenf** [sdv-dev/SDV#297]
### Environment Details

* SDV version: 0.5.0
* Python version: Python 3.7.9
* Operating System: Ubuntu 18.04.4 LTS (GNU/Linux 4.15.0-122-generic ppc64le)

### Error Description

The model HMA1 is not running even with the demo data. The script ends immediately, the training is not initiated. I tried both on the server and local macbook machine. Did not have such a problem with CopulaGANs.

### Steps to reproduce

I use the following code to reproduce the example given in the tutorial.

```
from sdv import Metadata
from sdv.relational import HMA1
from sdv import load_demo

metadata, tables = load_demo(metadata=True)
model = HMA1(metadata)
model.fit(tables)
```

**Comment by csala** [comment]
Hello @ThesisRepository 

I have a couple of questions:
1. Does it crash or print any error?
2. Have you tried running `model.sample()` after these lines?

Bear in mind that the demo data is very small, so the training of HMA1 in this dataset is almost immediate, and that there is no visible output during the training unless you have logging enabled, so I suspect that everything may actually be running as expected.

**Comment by elenagenf** [comment]
Hi @csala, 

Thank you for a prompt reply. Everything works, I expected to monitor training loss results but it is probably not feasible to implement in the case of relational tables generation.
